ProjectVoyage/View Controllers/Login and Signup/LoginVC.m

```diff
@@ -0,0 +1,65 @@
+//
+//  LoginVC.m
+//  ProjectVoyage
+//
+//  Created by Gui David on 7/6/22.
+//
+
+#import "LoginVC.h"
+@import Parse;
+
+@interface LoginVC ()
+@property (nonatomic, strong) IBOutlet UITextField *usernameField;
+@property (nonatomic, strong) IBOutlet UITextField *passwordField;
+@property (nonatomic, strong) IBOutlet UIButton *loginBtn;
+@property (weak, nonatomic) IBOutlet UIButton *signupBtn;
+
+@end
+
+@implementation LoginVC
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+
+
+- (IBAction)login:(id)sender {
+    NSString *username = self.usernameField.text;
+    NSString *password = self.passwordField.text;
+        
+    [PFUser logInWithUsernameInBackground:username password:password block:^(PFUser * user, NSError *  error) {
+        if (error != nil) {
+            [self alertFailedLogin];
+            NSLog(@"User log in failed: %@", error.localizedDescription);
+        } else {
+            NSLog(@"User logged in successfully");
+            // segue to home feed
+            [self performSegueWithIdentifier:@"loginSegue" sender:nil];
+        }
```

Comment:
do the strongself / weakself dance here (see example from PFQuery I put up earlier)